storage/category.go

```diff
@@ -8,9 +8,10 @@ import (
 
 // Category describes transaction category.
 type Category struct {
-	ID   int32  `json:"id"`
-	Name string `json:"name"`
-	Logo string `json:"logo"`
+	ID      int32  `json:"id"`
+	Visible bool   `json:"visible"`
```

Comment:
Are you sure that we need visible property? Maybe it will be better not to show specific category if it budget is 0?

Reply:
agreed
removed it